src/main/core/master.c

```diff
@@ -175,7 +175,13 @@ static gboolean _master_loadConfiguration(Master* master) {
             return FALSE;
         }
 
-        file = utility_getFileContents(fileName->str);
+        // Read config from file or stdin
+        if (0 == g_strcmp0("-", fileName->str)) {
+            file = utility_getFileContents("/dev/stdin");
```

Comment:
Maybe could I remove this condition by relocating `/dev/stdin` string in the `master->options` hydration ?

Reply:
I don't follow what you're suggesting, but this looks fine as is.